deepmd/infer/deep_pot.py

```diff
@@ -335,16 +354,17 @@ def eval(
             mixed_type=mixed_type,
         )
 
-        if self.modifier_type is not None:
-            if atomic:
-                raise RuntimeError("modifier does not support atomic modification")
-            me, mf, mv = self.dm.eval(coords, cells, atom_types)
-            output = list(output)  # tuple to list
-            e, f, v = output[:3]
-            output[0] += me.reshape(e.shape)
-            output[1] += mf.reshape(f.shape)
-            output[2] += mv.reshape(v.shape)
-            output = tuple(output)
+        if self.ntypes_spin == 0:
```

Comment:
Why do you need the condition here ?  It exclude the case: spin is used with modifier.

Reply:
We didn't use spin with modifier before. But now, we have included this case. But, it needs more tests.